x-pack/plugins/fleet/server/services/agents/reassign.ts

```diff
@@ -153,17 +189,28 @@ export async function reassignAgents(
     }))
   );
 
-  const orderedOut = givenOrder.map((agentId) => {
-    const hasError = agentId in outgoingErrors;
-    const result: BulkActionResult = {
+  let results;
+
+  if (!skipSuccess) {
```

Comment:
omitting successful agents from result to avoid hitting HTTP response limit (currently only for more than 10k actions)